src/pages/home/report/ReportActionCompose/AttachmentPickerWithMenuItems.js

```diff
@@ -130,10 +132,11 @@ function AttachmentPickerWithMenuItems({
     isFocused,
     raiseIsScrollLikelyLayoutTriggered,
 }) {
+    const actionSheetAwareScrollViewContext = useContext(ActionSheetAwareScrollView.ActionSheetAwareScrollViewContext);
     const theme = useTheme();
     const styles = useThemeStyles();
     const {translate} = useLocalize();
-    const {windowHeight} = useWindowDimensions();
+    const {windowWidth} = useWindowDimensions();
```

Comment:
just double checking, its correct we changed from height to width here?

Reply:
@hannojg I think it's correct, but actually @chrispader made these changes 🤔